js/exporter/jspdf/draw_utils.js

```diff
@@ -0,0 +1,87 @@
+import { isDefined } from '../../core/utils/type';
+import { drawTextInRect, drawRect } from './pdf_utils';
+import { extend } from '../../core/utils/extend';
+
+const defaultBorderLineWidth = 1;
+
+function drawRows(doc, rows, options) {
```

Comment:
At the 'paint' stage there will not be 'rows' or 'columns'
Instead, there will be 'list of rectangles' with texts, borders and background

Reply:
Yep, fixed. Now this method accepts only array[][] of pdfCells and has name drawPdfCells